R/boxes.R

```diff
@@ -278,25 +280,45 @@ box <- function(..., title = NULL, footer = NULL, status = NULL,
     titleTag <- h3(class = "box-title", title)
   }
 
+  boxToolsTag <- NULL
+  if (collapsible == TRUE || wrench == TRUE) {
+    boxToolsTag <- TRUE
+  } else {
+    boxToolsTag <- FALSE
+  }
+
   collapseTag <- NULL
-  if (collapsible) {
+  wrenchTag <- NULL
+
+  if (boxToolsTag) {
     buttonStatus <- status %OR% "default"
 
     collapseIcon <- if (collapsed) "plus" else "minus"
 
-    collapseTag <- div(class = "box-tools pull-right",
-      tags$button(class = paste0("btn btn-box-tool"),
-        `data-widget` = "collapse",
-        shiny::icon(collapseIcon)
+    if (collapsible == TRUE) {
+      collapseTag <- tags$button(class = paste0("btn btn-box-tool"), `data-widget` = "collapse", shiny::icon(collapseIcon))
+    }
+
+    if(wrench == TRUE) {
+      wrenchTag <- div(class = paste0("btn-group"),
+                       tags$button(class = "btn btn-box-tool dropdown-toggle", `type` = "button",
+                                   `data-toggle` = "dropdown", shiny::icon("wrench")),
+                       tags$ul(class = "dropdown-menu", `role` = "menu")
+                       ## TODO: how to display them ?
```

Comment:
Or of course: 
`wrenchOptions = tags$li(a(href="https://google.cz", "Click here!"))`
and wrenchOptions as an additional tag